ui/apps/platform/src/Containers/NetworkGraph/TopologyComponent.tsx

```diff
@@ -77,7 +77,6 @@ const TopologyComponent = ({
     selectedNode,
     simulator,
     setNetworkPolicyModification,
-    applyNetworkPolicyModification,
```

Comment:
Although style-checks is failing for a different reason, it is possible or even required
* remove from `TopologyComponentProps` above similar to in the following file?
* therefore also last import and type definition itself from userNetworkPolicySimulator hook?

Reply:
Should be good now